pageserver/src/config.rs

```diff
@@ -0,0 +1,671 @@
+//! Functions for handling page server configuration options
+//!
+//! Configuration options can be set in the pageserver.toml configuration
+//! file, or on the command line.
+//! See also `settings.md` for better description on every parameter.
+
+use anyhow::{anyhow, bail, ensure, Context, Result};
+use nix::NixPath;
```

Comment:
What's `nix::NixPath` for?

Reply:
Oh, good catch, it has a `PathBuf::is_empty()` method but I think ` == PathBuf::new()` is better since explicitly checks the default value of ours, I'll remove it.